### PR TITLE
feat: Avatar

### PR DESCRIPTION
**Pull request by gulam-mujtaba**
### First Component Added to Component Library.

**Features Added:**

1. Square-shaped Avatars with different sizes.
2. Rounded Avatars with different sizes.
3. Avatars with badges: square-shaped and round.

avatar ss

### CODE REVIEW

File: componenents/avatar.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
+@import url(/main.css);
+
+.all-containant {
```
Comment (zabihshaik) [inline componenents/avatar.css:3]: Better to use understandable class names

File: index.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
       <div class="navbar">
         <a href="#">Home</a>
         <a href="#">Features</a>
-        <a href="#">Documentation</a>
+        <a href="/componenents/avatar.html">Documentation</a>
```
Comment (zabihshaik) [inline index.html:28]: Typo for components

File: index.html [hunk 3]
```diff
@@ -40,13 +40,13 @@ <h2 class="heading">Welcome to DOV UI</h2>
           
         </p>
         <div class="btn-main">
-        <button class="start-btn">Get Started</button>
-        <button class="doc-btn">Documnetation</button>
```
Comment (zabihshaik) [inline index.html:44]: Typo for Documentation

**Review by zabihshaik — COMMENTED** [review]
Everything looks fine.
1. Check for Typos and also in naming Files
2. Better to add documentation to the components as well at the same time

File: index.html (hunk 2 above)
Comment (gulam-mujtaba) [reply index.html:28]: Thanks for the review. Will work on it.

**Pull request merged**